recipe/meta.yaml

```diff
@@ -37,10 +37,14 @@ test:
   requires:
     - pip
     - python {{ python_min }}
+    - pytest
   imports:
     - zarr
+  source_files:
+    - tests
```

Comment:
I'll be happy to find that I was wrong but I didn't think these were included in the distribution anymore. Let's see if this works.